scripts/averaging/create_TEM_files.py

```diff
@@ -102,18 +142,13 @@ def create_TEM_files(adf):
     if adf.get_basic_info("compare_obs"):
         print(f"\t Processing TEM for observations :")
 
-        output_loc_idx = output_loc / base_name
-        #Check if re-gridded directory exists, and if not, then create it:
-        if not output_loc_idx.is_dir():
-            print(f"    {output_loc_idx} not found, making new directory")
-            output_loc_idx.mkdir(parents=True)
-        #End if
-
         #Set baseline file name as full path
-        tem_fil = output_loc_idx / f'{base_name}.TEMdiag.nc'
+        tem_fil = tem_base_loc / f'{base_name}.TEMdiag.nc'
 
-        #Get baseline case tem over-write boolean
-        overwrite_tem = tem_opts.get("overwrite_tem_base")
+        #Get obs case tem over-write boolean
+        #QUESTION: should we make these and put in ADF obs location so we don't have to 
+        #worry about making these??
```

Comment:
If the obs data generally isn't changing then the answer to your question is YES.

Reply:
Giving it more thought, we should just remake these files each time in case the TEM obs files we have do change over time.